docs/content/CLIReference.md

```diff
@@ -104,6 +104,16 @@ Example:
 ./batect --skip-prerequisites the-task
 ```
 
+### Disable port binding on the host machine <small>(`--disable-ports`)</small>
+
+On CI systems or when running multiple batect tasks in parallel, host port binding will result in conflicts.
```

Comment:
It might be good to add some more context here, for example:

```suggestion
By default, batect binds ports from containers to the host machine running batect. These are defined with [`ports`](config/Containers.md#ports) on the container or [`ports`](config/Tasks.md#ports) on the task. 

However, in some situations, it can be useful to disable these bindings. In particular, when running multiple tasks in parallel, such as on CI systems, host port bindings can result in conflicts.

Passing `--disable-ports` disables all port bindings for all containers. This includes port bindings defined at the task level.
```

It might also be good to add a mention of this to `docs/content/tools/CIGeneral.md`.